Systems/c172p-hydrodynamics.xml

```diff
@@ -275,10 +275,10 @@
       </max>
      </product>
      <max>
-      <value>-5.0</value>
+      <value>-0.0</value>
       <min>
        <property>hydro/transverse-wave/squat-ft</property>
-       <value>5.0</value>
+       <value>0.0</value>
```

Comment:
Could you perhaps add a comment in the code that restricting to [-0.0, 0.0] is done in order to disable the squat because of the bobbing?